lib/sheet.js

```diff
@@ -35,15 +39,17 @@ const setCell = (target, prop, value) => {
   target.expressions.delete(prop);
   target.data.delete(prop);
 
-  if (typeof value === 'string' && value[0] === '=') {
-    const src = '() => ' + value.substring(1);
-    const options = { context: target.context };
-    const script = metavm.createScript(prop, src, options);
-    target.expressions.set(prop, { compute: script.exports, source: value });
-  } else {
-    target.data.set(prop, value);
+  try {
+    if (typeof value === 'string' && value[0] === '=') {
+      const src = '() => ' + value.substring(1);
+      const options = { context: target.context };
+      const script = metavm.createScript(prop, src, options);
+      target.expressions.set(prop, { compute: script.exports, source: value });
+    } else target.data.set(prop, value);
```

Comment:
If we use `{}` block for positive branch, negative one also should have `{}` block